### ISSUE TITLE
ICC Profile extraction

### ISSUE THREAD

**Issue by carlwilson** [openpreserve/jhove#110]
Hi,
I'm using JHove 1.9 and I want to validate ICC Profile from TIFF image created by scanner. Can JHove extract this information and add it into metadata?
I don't know TIFF standard very close. I want to extract CPP-CS-2012-1498 from following information:

```
00000500 00 07 57 2c 00 07 f9 01 43 49 45 44 00 07 57 2c |..W,....CIED..W,|
00000510 00 07 f9 01 64 65 73 63 00 00 00 00 00 00 00 11 |....desc........|
00000520 43 50 50 2d 43 53 2d 32 30 31 32 2d 31 34 39 38 |CPP-CS-2012-1498|
```

Could you help me please ?

@gmcgath commented on SourceForge:

> ICC Profile validation would certainly be a useful thing for JHOVE to do. It's a significant task, though, so it's not likely to happen unless it gets funding from somewhere.
> If anyone does want to undertake this project, of course, they're welcome to contribute. JHOVE is open source, after all.

**Comment by carlwilson** [comment]
This is identical to #61 and #62 which specify the issue in more detail. I'll leave it for now but please vote for one of the other issues.

**Comment by tledoux** [comment]
Should be closed by #249

**Comment by carlwilson** [comment]
Thanks very much @tledoux  :)
